src/server/mod.rs

```diff
@@ -96,6 +106,19 @@ impl<I> Server<I, ()> {
     }
 }
 
+impl Server<tokio_tcp::Incoming, ()> {
+    /// Create a new instance from a `std::net::TcpListener` instance.
+    #[cfg(feature = "runtime")]
```

Comment:
It seems CI noticed that this `cfg` should go on the `impl` instead.

Reply:
Actually, since this should change to convert into an `AddrIncoming`, this whole `impl` can just be merge with the one below it...